deployment/helm/resource-management-policies/balloons/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-resource-policy-balloons-config
```

Comment:
I think we want the helm labels here too

Reply:
Fixed, thanks